src/swap/components/SwapProvider.tsx

```diff
@@ -52,6 +57,17 @@ export function SwapProvider({
   const { useAggregator } = experimental;
   // Core Hooks
   const accountConfig = useConfig();
+  const [callsId, setCallsId] = useState<Hex>();
+  const callsStatus = useCallsStatus({
+    id: callsId || '0x',
+    query: {
+      refetchInterval: (query) => {
+        return query.state.data?.status === 'CONFIRMED' ? false : 1000;
+      },
+      enabled: callsId !== undefined,
+    },
+  });
+  const walletCapabilities = useCapabilitiesSafe({ chainId: base.id }); // Swap is only available on Base
```

Comment:
Do we want to force it like this? It feels a bit odd, like I will expect a `chainId` option at component level that defaults to `base.id` and than if it's not that there will be some logging explain what's going on.